### PR TITLE
added: homepage

### PR DESCRIPTION
**Pull request by sharmayatendra**
added: homepage
image
image

### CODE REVIEW

File: css/header.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    --badge-bg-color: rgb(255, 0, 0);
+}
+
+.header {
+    grid-area: header;
+    position: fixed;
+    padding: 6px;
+}
+
+span {
```
Comment (ooreally) [inline css/header.css:12]: Suggestion: use class instead of element for styling

Comment (ooreally) [inline css/header.css:12]: Suggestion: use classes instead of element for styling

**Review by ooreally — APPROVED** [review]
Looks good to merge.
Suggestion : use class instead of element for styling

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Pull request merged**